docs/integrations/guides/databricks.rst

```diff
@@ -0,0 +1,160 @@
+    :align: center
+    :scale: 30 %
+    :figclass: align-center
+
+**(Alternative 2 step 1) Storing the API Key in the AWS Secrets Manager**
+
+In the AWS management console ensure that your active region is the region you use for Databricks. Go to the *AWS Secrets Manager* and select *Store new secret*. Select *Other type of secrets* and add *api-key* as the key and paste the API key created in the previous step as the value. Click next.
+
+.. _hopsworks_secrets_manager.png: ../../_images/secrets_manager.png
+.. figure:: ../../imgs/cloud/secrets_manager.png
+    :alt: Hopsworks feature store secrets manager step 1
+    :target: `hopsworks_secrets_manager.png`_
+    :align: center
+    :scale: 20 %
+    :figclass: align-center
+
+As secret name enter *hopsworks/role/[MY_DATABRICKS_ROLE]* replacing [MY_DATABRICKS_ROLE] with the AWS role used by the Databricks instance that should access the Feature Store. Select next twice and finally store the secret. Then click on the secret in the secrets list and take note of the *Secret ARN*.
+
+.. _hopsworks_secrets_manager2.png: ../../_images/secrets_manager2.png
+.. figure:: ../../imgs/cloud/secrets_manager2.png
+    :alt: Hopsworks feature store secrets manager step 2
+    :target: `hopsworks_secrets_manager2.png`_
+    :align: center
+    :scale: 30 %
+    :figclass: align-center
+
+**(Alternative 2 step 2) Granting access to the secret to the Databricks notebook role**
+
+In the AWS management console go to *IAM*, select *Roles* and then the role that is used when creating Databricks clusters. Select *Add inline policy*. Choose *Secrets Manager* as service, expand the *Read* access level and check *GetSecretValue*. Expand Resources and select *Add ARN*. Paste the ARN of the secret created in the previous step. Click on *Review*, give the policy a name und click on *Create policy*.
+
+.. _hopsworks_aws_policy.png: ../../_images/aws_policy.png
+.. figure:: ../../imgs/cloud/aws_policy.png
+    :alt: Hopsworks feature store set policy
+    :target: `hopsworks_aws_policy.png`_
+    :align: center
+    :scale: 30 %
+    :figclass: align-center
+
+**Installing hopsworks-cloud-sdk**
+
+The feature store library needs to be installed to connect to it. In the Databricks UI, go to *Clusters* and select your cluster. Select *Libraries* and then *Install New*. As *Library Source* choose *PyPI* and fill in *hopsworks-cloud-sdk* into the *Package* field.
+
+**Creating a DBFS folder to store certificates**
+
+Hopsworks relies on certificates being available in the Databricks cluster to connect to some services inside Hopsworks. To ensure that these certificates can be distributed to all nodes in a Databricks cluster, Hopsworks relies on them being stored in the `Databricks file system <https://docs.databricks.com/data/databricks-file-system.html>`_.
+For this purpose, you should create a cert folder in DBFS. This can be done by running the following in a Databricks notebook connected to the prepared cluster::
```

Comment:
cert folder -> certs folder
running the following -> executing the following statement